src/js/game/systems/display.js

```diff
@@ -94,4 +94,50 @@ export class DisplaySystem extends GameSystemWithFilter {
             }
         }
     }
+
+    /**
+     * Draws overlay of a given chunk
+     * @param {import("../../core/draw_utils").DrawParameters} parameters
+     * @param {MapChunkView} chunk
+     */
+    drawChunkOverlay(parameters, chunk) {
+        const contents = chunk.containedEntitiesByLayer.regular;
+        for (let i = 0; i < contents.length; ++i) {
+            const entity = contents[i];
+            if (!entity || !entity.components.Display) {
+                continue;
+            }
+
+            const pinsComp = entity.components.WiredPins;
+            const network = pinsComp.slots[0].linkedNetwork;
+
+            if (!network || !network.currentValue) {
+                continue;
+            }
+
+            const value = this.getDisplayItem(network.currentValue);
+
+            if (!value) {
+                continue;
+            }
+
+            const origin = entity.components.StaticMapEntity.origin;
+            if (value instanceof ColorItem) {

```

Comment:
use `value.getItemType() === "color"`

Reply:
instanceof makes more sense from oop point, and using `.getItemType` breaks lint -- you can make a webpack define that converts `instanceof *Item` to these calls, though.